docs/prometheus.md

```diff
@@ -9,245 +9,361 @@ sidebar_label: Prometheus
 
-- Start with small storage and expand the size of TSDB as needed on the fly
 
-- Prometheus metrics are highly available. When a node fails or rebooted, Prometheus pod is rescheduled onto on of the two other nodes where cStor volume replica is available. The metrics data is  rebuilt when the node becomes available
 
-- Take backup of the Prometheus metrics periodically and back them up to S3 or any object storage so that restoration of the same metrics is possible to the same or any other Kubernetes cluster
+We will add 100G of two disks to each node. Disks will be consumed by Prometheus and Alert manager instances using OpenEBS local PV device storage engine. The recommended configuration is to have at least three nodes and two unclaimed external disks to be attached per node. 
 
 
-<br>
 
-<hr>
+## Configuration workflow
 
-<br>
+1. [Install OpenEBS](/docs/next/prometheus.html#install-openebs)
 
-## Deployment model 
+2. [Select OpenEBS storage engine](/docs/next/prometheus.html#select-openebs-storage-engine)
 
-<br>
+3. [Configure OpenEBS Local PV StorageClass](/docs/next/prometheus.html#configure-openebs-local-pv-storageclass)
 
-<img src="/docs/assets/svg/prometheus-deployment.svg" style="width:100%;">
+4. [Installing Prometheus Operator](/docs/next/prometheus.html#installing-prometheus-operator)
 
-As shown above, OpenEBS volumes need to be configured with three replicas for high availability. This configuration work fine when the nodes (hence the cStor pool) is deployed across Kubernetes zones.
+5. [Accessing Prometheus and Grafana](/docs/next/prometheus.html#accessing-prometheus-and-grafana)
 
+   
 
+### Install OpenEBS
 
-<br>
+If OpenEBS is not installed in your K8s cluster, this can be done from [here](/docs/next/installation.html). If OpenEBS is already installed, go to the next step.
 
-<hr>
 
-<br>
 
-## Configuration workflow
+### Select OpenEBS storage engine
 
-<br>
+A storage engine is the data plane component of the IO path of a Persistent Volume. In CAS architecture, users can choose different data planes for different application workloads based on a configuration policy. OpenEBS provides different types of storage engines and chooses the right engine that suits your type of application requirements and storage available on your Kubernetes nodes. More information can be read from [here](/docs/next/overview.html#types-of-openebs-storage-engines).
+
+After OpenEBS installation, choose the OpenEBS storage engine as per your requirement. 
+
+- Choose **cStor**, If you are looking for replicated storage feature and other enterprise graded features such as volume expansion, backup and restore, etc. The steps for Prometheus operator installation using OpenEBS cStor storage engine can be obtained from [here](https://github.com/openebs/cstor-operators/blob/master/docs/workload/prometheus-operator/prometheus.md).
+- Choose **OpenEBS Local PV**, if you are not requiring replicated storage but high performance storage engine. 
```

Comment:
if you do not require durable metrics, and primarily use Prometheus for generating alerts.